city_scrapers/spiders/chi_ssa_60.py

```diff
@@ -0,0 +1,129 @@
+import html
+import json
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE, COMMISSION, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+from scrapy import Request
+
+
+class ChiSsa60Spider(CityScrapersSpider):
+    name = "chi_ssa_60"
+    agency = "Chicago Special Service Area #60 Albany Park"
+    timezone = "America/Chicago"
+    months_backward = 10
+    months_forward = 1
+
+    start_urls = None  # Generated by `start_requests`
```

Comment:
If you're using `start_requests` I think you can remove this

Reply:
Good point.